chains/manager.go

```diff
@@ -976,7 +948,7 @@ func (m *manager) createAvalancheChain(
 		Avalanche: &handler.Engine{
 			StateSyncer:  nil,
 			Bootstrapper: avalancheBootstrapper,
-			Consensus:    avalancheEngine,
+			Consensus:    nil,
```

Comment:
This is currently broken. This doesn't support bootstrapping the DAG after bootstrapping has finished.